index.html

```diff
@@ -941,8 +941,12 @@ <h1><dfn>Coalesced and predicted events</dfn></h1>
             <h2><dfn>Coalesced events</dfn></h2>
 
             <p>For performance reasons, user agents may choose not to send a <code>pointermove</code>
-            event every time the position of a pointer is updated. Instead, they will coalesce
-            (combine/merge) multiple position changes into a single move event. While
+            event every time a <a data-lt="measurable properties">measurable property</a>
+            (such as coordinates, pressure, tangential pressure, tilt, twist, or contact geometry) or a
+            property relating to keyboard modifiers (<code>ctrlKey</code>, <code>shiftKey</code>,
+            <code>altKey</code>, and <code>metaKey</code> from the [[UI-EVENTS]] Mouse Event model)
```

Comment:
Because no pointer event is fired for changes in `*Key` modifiers (see the comment in the glossary), let's drop them from here too.  (Sorry again for suggesting them initially).

Reply:
doh, you're right, no need to even mention them here anymore.